vite.config.mts

```diff
@@ -25,6 +25,7 @@ export default defineConfig({
     jsx: 'automatic',
   },
   optimizeDeps: {
+    exclude: ['storybook'],
```

Comment:
Can you try removing cache (`node_modules/.cache`) and clean install of deps?

I'm not able to reproduce the issue that made you add this.

Reply:
Just `npm ci`. It will remove `node_modules`, including the `.cache` subfolder.